ecl2df/wellcompletiondata.py

```diff
@@ -0,0 +1,248 @@
+"""Aggregates completion data from layer to zone"""
```

Comment:
A thought: Should the module be renamed, if it always aggregates to zones? Otherwise I would head here to get all possible data on wellcompletions (which are cell by cell), but would be disappointed if it could not give me individual well completions?

Reply:
Yes, names are always difficult, I have been trying to make a distinction between _connection_ (which is per cell, like in the wellconnstatus module) and _completion_ that is something more aggregated. But what could be a good name: _zonecompletion_ ?

Reply:
Ah, so if we just step outside the discretized world of the current grid, "a single completion" is maybe more thought of relating to one zone. Yes, keep it.

Reply:
And it is also consistent with webviz where the plugin is called WellCompletion and it vizualises the data per zone. Good, I'll keep it then.